backend/typescript/middlewares/validators/isValid.ts

```diff
@@ -0,0 +1,25 @@
+type Type = "string" | "integer" | "boolean" | "string-array";
+
+const isValid = (value: any, type: Type): boolean => {
+  if (value === undefined || value === null) return false;
+
+  if (type === "string") {
+    return typeof value === "string";
+  }
+  if (type === "boolean") {
+    return typeof value === "boolean";
+  }
+  if (type === "integer") {
+    return typeof value === "number" && Number.isInteger(value);
+  }
+  if (type === "string-array") {
```

Comment:
This is just a nice-to-have but we could create separate `validatePrimitive` and `validateArray` functions, where `validateArray` is passed the type of its elements. This way we can easily validate arrays of any type. Up to you if you want to make this change now, if not, I can create a task for the future